mitodl/ocw-to-hugo
# Linting and testing

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-to-hugo/issues/6

#### What's this PR do?
Adds a basic linting and testing configuration to the project, as well as the Github templates

#### How should this be manually tested?
Run `npm test`, `npm run lint` and `npm run fmt:check` and there should be no errors
